vitess-21.0.yaml

```
@@ -0,0 +1,262 @@
+          mkdir -p "${{targets.contextdir}}"/vt/config
+          mkdir -p "${{targets.contextdir}}"/vt/vtdataroot
+          # symlink only ${KEEP_BINARIES}
+          for bin in ${KEEP_BINARIES}; do
+            ln -sf /usr/bin/$bin "${{targets.contextdir}}"/vt/bin/
+          done
+    test:
+      environment:
+        contents:
+          packages:
+            - vitess
+      pipeline:
+        - name: Ensure symlinks are valid
+          runs: |
+            #!/bin/bash
+            find /vt/bin -type l ! -exec test -e {} \; -print
+
+  - name: "${{package.name}}-binaries"
+    description: "All the binaries built by Vitess"
+    dependencies:
+      provides:
+        - vitess-binaries=${{package.full-version}}
+    pipeline:
+      - name: Copy all binaries
+        runs: |
+          mkdir -p ${{targets.contextdir}}/usr/bin
+          cp -r ${{targets.destdir}}/usr/bin/* ${{targets.contextdir}}/usr/bin
+      - name: Remove redundant binaries
+        runs: |
+          for bin in $(ls ${{targets.destdir}}/usr/bin); do
+            if [[ ! " ${KEEP_BINARIES} " =~ " ${bin} " ]]; then
+              rm -f ${{targets.destdir}}/usr/bin/${bin}
+            fi
+          done
+    test:
+      pipeline:
+        - name: Smoke test for vitess binaries
+          runs: |
+            autogenerate --version
+            autogenerate --help
+            demo --version
+            demo --help
+            docgen --version
+            docgen --help
+            example --help
+            go-upgrade --help
+            maketestdata --help
+            mysqlctl --version
+            mysqlctl --help
+            mysqlctld --version
+            mysqlctld --help
+            rulesctl --version
+            rulesctl --help
+            topo2topo --version
+            topo2topo --help
+            vtaclcheck --version
+            vtaclcheck --help
+            vtadmin --version
+            vtadmin --help
+            vtbackup --version
+            vtbackup --help
+            vtbench --version
+            vtbench --help
+            vtclient --version
+            vtclient --help
+            vtcombo --version
+            vtcombo --help
+            vtctl --version
+            vtctl --help
+            vtctlclient --version
+            vtctlclient --help
+            vtctld --version
+            vtctld --help
+            vtctldclient --version
+            vtctldclient --help
+            vterrorsgen --version
+            vterrorsgen --help
+            vtexplain --version
+            vtexplain --help
+            vtgate --version
+            vtgate --help
+            vtgateclienttest --help
+            vtorc --version
+            vtorc --help
+            vttablet --version
+            vttablet --help
+            vttestserver --version
+            vttestserver --help
+            vttlstest --help
+
+update:
+  enabled: true
+  github:
+    identifier: vitessio/vitess
+    strip-prefix: v
+    use-tag: true
+    tag-filter: v20.
```

I always do a quick skim of the tagging configuration, and looks like the automation didn't bump the tag filter:

```suggestion
    tag-filter: v21.
```

---

I'm surprised the CI checks passed and didn't complain about not being able to find the tag version

---

Great catch, thank you! Just pushed up a fix.